azure/azure-iot-hub-device-simulator/src/main/resources/logback.xml

```diff
@@ -1,4 +1,4 @@
-<configuration debug="false" scan="false" scanPeriod="30 seconds">
+<configuration debug="false">
```

Comment:
Is `scan=false` the default? If not I suggest re-add it. The `scanPeriod` can be omitted.